_data/toc.yaml

```diff
@@ -1645,6 +1645,14 @@ manuals:
           title: Build contexts and linking targets
         - path: /build/bake/compose-file/
           title: Building from Compose file
+    - sectiontitle: Attestations
+      section:
+        - path: /build/attestations/
+          title: Overview
+        - path: /build/attestations/provenance/
+          title: Provenance
+        - path: /build/attestations/sbom/
+          title: SBOM
```

Comment:
I think SBOMs should appear before the Provenance section, since I think if users are familiar with only one of these, it's likely the former. 

```suggestion
        - path: /build/attestations/sbom/
          title: SBOM
        - path: /build/attestations/provenance/
          title: Provenance
```